badges/badge.css

```diff
@@ -0,0 +1,178 @@
+@import url(/CSS/nav.css);
+
+
+:root {
+
+    --green-badge: #16a34a;
```

Comment:
no need to repeat  :root in every file make 
keep it in a single file and access from there

Reply:
Thank you, Mimansha. I missed adding position: relative to the parent container and hence the badges were moving all around the place. I'll look into the other suggestions as well.